packages/framework/src/Model/Pricing/BasePriceCalculation.php

```diff
@@ -41,25 +41,6 @@ public function calculateBasePrice($inputPrice, $inputPriceType, Vat $vat)
         return new Price($basePriceWithoutVat, $basePriceWithVat);
     }
 
-    /**
-     * @param \Shopsys\FrameworkBundle\Model\Pricing\Price $price
-     * @param \Shopsys\FrameworkBundle\Model\Pricing\Vat\Vat $vat
-     * @param string[] $coefficients
-     * @return \Shopsys\FrameworkBundle\Model\Pricing\Price
-     */
-    public function applyCoefficients(Price $price, Vat $vat, $coefficients)
```

Comment:
short answer: YES
long answer: If You have in mind use-case that will require this method soon, then we have to consider keeping this method. But if it is just "useful", then I'm strongly for removing such code. Imagine in one year one will look at this code and ask "why is it here?", then no one will know. This reminds me commented out code that might be useful in future - it never is.

Reply:
And I agreed with @vitek-rostislav that we want to remove unused code right now (as we are in β)

Reply:
I know that `applyCoefficients` is useful for applying discounts and it was used on implemented projects. Since it is unit tested, public and useful, I'm quite against removal of this particular method. Even so, I'd be fine with it if you both agree that it should be removed and it is explicitly mentioned and agreed upon during business validation.

Reply:
in the end, we keep `applyCoefficients`